gocourse03/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"math/rand/v2"
+	"slices"
+	"time"
+)
+
+func main() {
+	rodents := []Rodent{
+		NewRodent(11, RodentRat, FromTo{}, NewDailyMovements()),
+		NewRodent(12, RodentRat, FromTo{}, NewDailyMovements()),
+		NewRodent(13, RodentRat, FromTo{}, NewDailyMovements()),
+		NewRodent(66, "To delete type", FromTo{}, NewDailyMovements()),
+	}
+
+	rodents = slices.Delete(rodents, 3, 4)
+
+	startMovement := time.Now()
+	for key, _ := range rodents {
```

Comment:
Спробуй, чи ось так працюватиме:
```suggestion
	for key := range rodents {
```